src/features/island/collectibles/components/pets/PetFeed.tsx

```diff
@@ -0,0 +1,263 @@
+              <div className="flex flex-row gap-2 items-center">
+                <img src={xpIcon} className="w-4" />
+                <div className="flex flex-row md:items-center gap-1 text-xs">
+                  <span>
+                    {`${currentProgress} / ${experienceBetweenLevels}`}
+                  </span>
+                </div>
+              </div>
+              <div className="flex flex-row items-center gap-2">
+                <img src={SUNNYSIDE.icons.lightning} className="w-3" />
+                <span className="text-xs font-medium">{`${beforeEnergy}`}</span>
+              </div>
+            </InnerPanel>
+            <div className="w-[20%] sm:w-full flex justify-center items-center">
+              <img
+                src={SUNNYSIDE.icons.arrow_right}
+                alt="Arrow Right"
+                className="w-4 block md:hidden"
+              />
+              <img
+                src={SUNNYSIDE.icons.arrow_down}
+                alt="Arrow Down"
+                className="w-4 hidden md:block"
+              />
+            </div>
+            <InnerPanel className="flex flex-col gap-2 w-[40%] sm:w-full">
+              <div className="flex flex-row items-center gap-2">
+                <img src={levelUp} className="w-3" />
+                <span className="text-xs font-medium">{`Level: ${levelAfterFeed} ${levelChange > 0 ? `(+${levelChange})` : ""}`}</span>
+              </div>
+              <div className="flex flex-row gap-2 items-center">
+                <img src={xpIcon} className="w-4" />
+                <div className="flex flex-row md:items-center gap-1 text-xs">
+                  <span>
+                    {`${currentProgressAfterFeed} / ${experienceBetweenLevelsAfterFeed} ${experienceChange > 0 ? `(+${experienceChange})` : ""}`}
+                  </span>
+                </div>
+              </div>
+              <div className="flex flex-row items-center gap-2">
+                <img src={SUNNYSIDE.icons.lightning} className="w-3" />
+                <span className="text-xs font-medium">{`${afterEnergy} ${energyChange > 0 ? `(+${energyChange})` : ""}`}</span>
+              </div>
+            </InnerPanel>
+          </div>
+        </div>
+      ) : (
+        <div className="flex flex-row md:flex-col gap-2 justify-between w-full p-1">
+          <div className="flex flex-row gap-2 items-center">
+            <img src={xpIcon} className="w-4" />
+            <span className="text-xs">{`+${foodXp} XP`}</span>
+          </div>
+          <div className="flex flex-row gap-2">
+            <img src={SUNNYSIDE.icons.lightning} className="w-3" />
+            <span className="text-xs">{`+${foodXp} Energy`}</span>
+          </div>
+        </div>
+      )}
+
+      {/* Labels for today's feed and insufficient food */}
+      {isToday && foodFed?.includes(selectedFood) ? (
+        <div className="flex w-full items-start">
+          <Label type="danger" className="text-xs">
```

Comment:
<img width="320" height="192" alt="Screenshot 2025-09-02 at 6 40 13 PM" src="https://github.com/user-attachments/assets/3cae354c-f09a-46c3-a796-7673eeda5534" />

It wasn't clear which foods had already been completed for the day. I had to click on each request to see if I'd completed it or not